system-tests/projects/runner-e2e-specs/cypress/e2e/errors/readfile.cy.js

```diff
@@ -1,7 +1,5 @@
-import './setup'
-
 describe('cy.readFile', () => {
   it('existence failure', () => {
-    cy.readFile('does-not-exist', { timeout: 0 })
```

Comment:
Why did this timeout need increased?

Reply:
Good question. With a timeout of 0, the command would sometimes not report why the timeout occurred: 'the command readFile timed out with a timeout of 0' vs. 'the command timed out with a timeout of 0 because the file does not exist'. Increasing the timeout slightly results in the latter result consistently, which is the behavior being asserted in the original test. I assume this is a quirk in the timing between the old and new cypress-in-cypress  setups, but haven't dug much deeper. I also think a timeout greater than zero is more representative in general 🤷‍♂️ but I'll double check this again to see what might be different.

Reply:
Digging done. With a timeout of 0, there is a chance that the backend request will time out before it can fail with an existence error, which is where this discrepancy comes from. But it's very inconsistent, less than 1 out of 20 runs. Bumping it from a non-zero value mitigates this race condition. That doesn't explain why this wasn't flaky before, but we are changing how this code executes quite a bit...